diff --git a/AjaxControladores/menu.ajax.controlador.php b/AjaxControladores/menu.ajax.controlador.php
index fde6481..98095d1 100644
--- a/AjaxControladores/menu.ajax.controlador.php
+++ b/AjaxControladores/menu.ajax.controlador.php
@@ -39,8 +39,6 @@
     }
     
     if(isset($_POST["contador_productos"])){
-        echo "a ver: ".$_POST["txt_cantidad"];
-        echo "a ver: ".$_POST["txt_idProducto"];
         $contador_producto = new ajaxMenu();
         $contador_producto->cantidad_productos = $_POST["contador_productos"];        
         $contador_producto->ajaxContadorProductos();
diff --git a/AjaxControladores/mesa.ajax.controlador.php b/AjaxControladores/mesa.ajax.controlador.php
index 4e72495..e5aa339 100644
--- a/AjaxControladores/mesa.ajax.controlador.php
+++ b/AjaxControladores/mesa.ajax.controlador.php
@@ -3,6 +3,7 @@
     require_once '../Modelo/mesa.modelo.php';
     
     class MesaAjaxControlador{
+        public $nro_mesa;
         public function ctrAjaxAgregarMesa(){
             $tabla = "mesa";
             $resultado = MesaModelo::agregarMesa($tabla);
@@ -19,11 +20,28 @@
         }
     }
     
+    
+    if(isset($_POST["contador_productos"])){
+        $contador_producto = new ajaxMenu();
+        $contador_producto->cantidad_productos = $_POST["contador_productos"];        
+        $contador_producto->ajaxContadorProductos();
+    }
+    
         
     if($_SERVER['REQUEST_METHOD'] === 'POST'){
-	$mesa = new MesaAjaxControlador();
+        
+        $mesa = new MesaAjaxControlador();
         $mesa->ctrAjaxAgregarMesa();
-    } 
+//        if(isset($_POST['id'])){
+//            $mesa = new MesaAjaxControlador();
+//            $mesa->nro_mesa = $_POST['id'];
+//            $mesa->
+//            
+//        }else{
+//            $mesa = new MesaAjaxControlador();
+//            $mesa->ctrAjaxAgregarMesa();
+//        }	
+    }
     
 ?>
 
diff --git a/AjaxControladores/mesa.datatable.controlador.php b/AjaxControladores/mesa.datatable.controlador.php
index 3c9ff33..f037a61 100644
--- a/AjaxControladores/mesa.datatable.controlador.php
+++ b/AjaxControladores/mesa.datatable.controlador.php
@@ -2,13 +2,25 @@
     include_once '../Controlador/mesa.controlador.php';
     require_once '../Modelo/mesa.modelo.php';
 
-
 class TablaMesa {
         public $nroMesa;
         public function verDetalleMesa(){
             $nm = $this->nroMesa;
             $respuesta = MesaControlador::ctrVerDetalleMesa($nm);
-            return $respuesta;
+            
+            $tablaDetalleMesa = "";
+            
+            foreach($respuesta as $key => $value){
+                $tablaDetalleMesa .= '{
+                                        "item" : "'.($key+1).'",
+                                        "plato" : "'.$value["nombre_producto"].'",
+                                        "cantidad" : "'.$value["cantidad"].'",
+                                        "total" : "'.$value["precio"].'"
+                                    },';
+            }
+               
+            $tablaDetalleMesa = substr($tablaDetalleMesa, 0, strlen($tablaDetalleMesa)-1);
+            echo '{"data":['.$tablaDetalleMesa.']}';            
         }
     }
 
diff --git a/Controlador/mesa.controlador.php b/Controlador/mesa.controlador.php
index 801350c..fa69d8c 100644
--- a/Controlador/mesa.controlador.php
+++ b/Controlador/mesa.controlador.php
@@ -8,7 +8,7 @@
         }
         public static function ctrVerDetalleMesa($nro_mesa){
             $resultado = MesaModelo::verDetalleMesa($nro_mesa);
-            echo $resultado;
+            return $resultado;
         }
     }
   
diff --git a/Modelo/mesa.modelo.php b/Modelo/mesa.modelo.php
index 5a5f4f6..b289596 100644
--- a/Modelo/mesa.modelo.php
+++ b/Modelo/mesa.modelo.php
@@ -39,16 +39,18 @@
             $ultimaMesa = MesaModelo::ultimaMesaAgregada($tabla) + 1;
             $consulta = "INSERT INTO $tabla(numero_mesa) values($ultimaMesa)";
             $statement = Conexion::Conectar()->prepare($consulta);            
-            $statement->execute();      
-            //$statement -> close();
-            //$statement = null;
+            $statement->execute();
+            
+            $statement -> close();
+            $statement = null;
         }
 
         public function verDetalleMesa($numeroMesa) {
             $consulta = "SELECT producto.nombre_producto, 
                                 pedido_producto.cantidad, 
                                 personal.nombre_personal, 
-                                pedido.sub_total 
+                                producto.precio,
+                                pedido.sub_total
                         FROM mesa
                         INNER JOIN pedido
                         ON mesa.id_mesa = pedido.MESA_id_mesa
@@ -63,6 +65,8 @@
                         WHERE mesa.numero_mesa = '$numeroMesa'";
             $statement = Conexion::Conectar()->prepare($consulta);
             $statement->execute();
+            
+            
             $pedidos = $statement->fetchAll(PDO::FETCH_CLASS);
             $i = 1;
             $aux = [];
@@ -80,7 +84,13 @@
             $retornoPedidos = json_encode([
                 "data" => $aux
             ]);
-            return $retornoPedidos;
+            
+            
+            
+            return $statement->fetchAll();
+            
+            $statement -> close();
+            $statement = null;
         }
     }
     
diff --git a/Vista/js/mesa.js b/Vista/js/mesa.js
index 3e6d124..28a7475 100644
--- a/Vista/js/mesa.js
+++ b/Vista/js/mesa.js
@@ -36,8 +36,10 @@ $('#agregarMesa').on('click', function(){
     agregarMesa();
 });
 
+var mesa;
+
 function verDetalleMesa(nro_mesa){
-    var mesa = {"nro_mesa" : nro_mesa};
+    mesa = {"nro_mesa" : nro_mesa};
     $('#lbl_nroMesa').text('Mesa N°: ' + nro_mesa);
     $('.tabla_verDetalleMesa').DataTable().destroy();
     $('.tabla_verDetalleMesa').DataTable({
@@ -52,7 +54,6 @@ function verDetalleMesa(nro_mesa){
                         {"data":"item"},
                         {"data":"plato"},
                         {"data":"cantidad"},
-                        {"data":"nombre_mozo"},
                         {"data":"total"}
                     ],
         "language":    {
@@ -80,12 +81,20 @@ function verDetalleMesa(nro_mesa){
                             }
                         }
     });
+//    debugger;
+//     $.ajax({
+//        url:"AjaxControladores/mesa.datatable.controlador.php",
+//      	method: "POST",
+//      	data: mesa,
+//      	dataType:"json",
+//      	success:function(respuesta){
+//            $('#lbl_totalMesa').html(respuesta['sub_total']);
+//            $('#lbl_totalMesa').val(respuesta['sub_total']);
+//        }
+//    });
 }
 
 
-
-
-
 //    $('#agregarMesa').on('click',function(){
 //        
 //    });
@@ -105,7 +114,6 @@ function verDetalleMesa(nro_mesa){
 
 
 
-
 //            var imagen_mesa = $('#mesa');
 ////            //var nueva_imagen_mesa = '<img class="" src="Vista/img/contenido/mesa.png"/>';
 ////
@@ -123,3 +131,15 @@ function verDetalleMesa(nro_mesa){
 ////                imagen_mesa.append(nueva_imagen_mesa);
 ////            });
 
+function cerrarMesa(){
+    var datos = {"id": mesa};
+    $.ajax({
+        url:"AjaxControladores/mesa.ajax.controlador.php",
+      	method: "POST",
+      	data: datos,
+      	dataType:"json",
+      	success:function(respuesta){
+            
+        }
+    });
+}
\ No newline at end of file
diff --git a/Vista/js/pedido.js b/Vista/js/pedido.js
index eb36313..a7a378b 100644
--- a/Vista/js/pedido.js
+++ b/Vista/js/pedido.js
@@ -172,8 +172,9 @@ function sumarPreciosProductosSeleccionados(){
     $('#totalPrecioProducto').val(sumaTotalPrecio.toFixed(2));
 }
 
-//
+//CREAR PEDIDO
 $('#crearVenta').on('click', function(){
+    debugger;
     //Para insertar en tabla Pedido
     var idMozo = $("#seleccioneEmpleado").val();
     var idMesa = $("#seleccioneMesa").val();
diff --git a/Vista/modulos/contenido/mesas.php b/Vista/modulos/contenido/mesas.php
index 8665b11..77de276 100644
--- a/Vista/modulos/contenido/mesas.php
+++ b/Vista/modulos/contenido/mesas.php
@@ -45,7 +45,7 @@
 MODAL PARA VER MESA
 ------------------------------------------->
 <div id="modalVerDetalleMesa" class="modal fade" role="dialog">
-    <div class="modal-dialog">
+    <div class="modal-dialog modal-lg">
         <div class="modal-content">
             <form role="form" method="POST">
                 <!--=====================================
@@ -61,11 +61,11 @@ MODAL PARA VER MESA
                 <div class="modal-body">
                     <div class="box-body">
                         <div class="row">
-                            <div class="col-md-6">
+                            <div class="col-md-7">
                                 <a href="pedido" class="btn btn-success"><i class="fa fa-plus"></i>&nbsp;Agregar Pedido</a>
                             </div>
-                            <div class="col-md-6 row">
-                                <label>TOTAL:&nbsp;</label><label id="lbl_totalMesa" style="font-size: 16px;"></label>
+                            <div class="col-md-5 row">
+                                <label style="font-size: 16px;">TOTAL:&nbsp;</label><label id="lbl_totalMesa" style="font-size: 16px;">holi</label>
                             </div>
                         </div>
                         <br>
@@ -74,8 +74,7 @@ MODAL PARA VER MESA
                                 <tr>
                                     <th style="width: 10px">N°</th>
                                     <th style="width: 100px">Plato</th>
-                                    <th style="width: 150px">Cantidad</th> 
-                                    <th style="width: 100px">Mozo</th>
+                                    <th style="width: 150px">Cantidad</th>
                                     <th style="width: 100px">Total</th>
                                 </tr>
                             </thead>
@@ -87,6 +86,7 @@ MODAL PARA VER MESA
                 ======================================-->
                 <div class="modal-footer">
                     <button type="button" class="btn btn-default pull-left" data-dismiss="modal">Salir</button>
+                    <button class="btn bg-navy pull-right" onclick="cerrarMesa()">Cerrar Mesa</button>
                 </div>
             </form>
         </div>
diff --git a/nbproject/private/private.xml b/nbproject/private/private.xml
index 2f89553..8dbbddc 100644
--- a/nbproject/private/private.xml
+++ b/nbproject/private/private.xml
@@ -20,10 +20,14 @@
     </editor-bookmarks>
     <open-files xmlns="http://www.netbeans.org/ns/projectui-open-files/2">
         <group>
-            <file>file:/C:/xampp/htdocs/PortonAzulv1.0/Vista/modulos/contenido/mesas.php</file>
-            <file>file:/C:/xampp/htdocs/PortonAzulv1.0/Modelo/mesa.modelo.php</file>
-            <file>file:/C:/xampp/htdocs/PortonAzulv1.0/Controlador/mesa.controlador.php</file>
-            <file>file:/C:/xampp/htdocs/PortonAzulv1.0/Vista/js/mesa.js</file>
+            <file>file:/C:/xampp/htdocs/porton-azul-repo/AjaxControladores/personal.datatable.ajax.php</file>
+            <file>file:/C:/xampp/htdocs/porton-azul-repo/AjaxControladores/mesa.datatable.controlador.php</file>
+            <file>file:/C:/xampp/htdocs/porton-azul-repo/Controlador/mesa.controlador.php</file>
+            <file>file:/C:/xampp/htdocs/porton-azul-repo/Vista/js/pedido.js</file>
+            <file>file:/C:/xampp/htdocs/porton-azul-repo/Vista/js/mesa.js</file>
+            <file>file:/C:/xampp/htdocs/porton-azul-repo/Modelo/mesa.modelo.php</file>
+            <file>file:/C:/xampp/htdocs/porton-azul-repo/Modelo/gestion_usuario.modelo.php</file>
+            <file>file:/C:/xampp/htdocs/porton-azul-repo/Vista/modulos/contenido/mesas.php</file>
         </group>
     </open-files>
 </project-private>
